recommendation/dlrm_v2/pytorch/docker_gpu/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM nvidia/cuda:11.0-cudnn8-devel-ubuntu18.04
```

Comment:
Looks like this image doesn't exist on the NVIDIA docker hub (https://hub.docker.com/r/nvidia/cuda/tags?page=1&name=11.0)

Changing it from `11.0` to `11.0.3` seems to work.

Reply:
Seconded^^ Can this be changed to a Ubuntu 20 or 22.04 image?